atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/async/query/GetQuerySpec.java

```diff
@@ -47,40 +48,46 @@
             + "ORDER BY column1, column2 ASC "
             + "LIMIT 1;";
 
-    @org.immutables.value.Value.Auxiliary
-    public abstract ByteBuffer row();
-
-    @org.immutables.value.Value.Auxiliary
-    public abstract ByteBuffer column();
+    @Override
+    public String formatQueryString() {
```

Comment:
In the format string, do you need the ORDER BY? it's a limit 1 anyway, and also it's ordering by things you're not even selecting.

Reply:
The `ORDER BY` was added more for documenting purposes, the data is already stored in C* by the order specified there so this ORDER BY to actually be a NO OP during execution of the query. I will remove it.

Reply:
For completeness, data is returned in order of clustering key given a partition key is provided, so yeah `ORDER BY` isn't needed. (Note that this query if interpreted as SQL for another databases would likely require the ORDER BY.)